Linked_List/StartOfLoop.java

```diff
@@ -0,0 +1,94 @@
+import java.util.Scanner;
+
+class StartOfLoop
+{
+	static Node head;
+	static class Node
+	{
+		int data;
+		Node next;
+		Node(int d)
+		{
+			data = d;
+			next = null;
+		}
+	}
+
```

Comment:
Give 4 space indentation across the file

Reply:
I have made the required changes.